python/ray/serve/api.py

```diff
@@ -235,11 +238,11 @@ def get_backend_config(backend_tag):
 
 
 @_ensure_connected
-def create_backend(backend_tag,
-                   func_or_class,
+def create_backend(backend_tag: str,
+                   func_or_class: Callable,
                    *actor_init_args,
                    ray_actor_options=None,
-                   config=None):
+                   config: Dict[str, Any] = None) -> None:
```

Comment:
`Optional[Dict[str,Any]]`

Reply:
All fixed, thanks for the careful check!